### ISSUE TITLE
Flair Filter is not working

### ISSUE THREAD

**Issue by JourneyOver** [honestbleeps/Reddit-Enhancement-Suite#5028]
- My browser (copy from https://www.whatismybrowser.com): Chrome 70 on Windows 10
- My RES version: v5.13.5
- Am I in reddit beta (yes/no): No

-- other information

Have tried latest version of chrome as well clean install no other extensions other than RES, I have nightmode enabled, and I have cookies enabled.

*What's the problem?*

Flair filters are not working, no matter the subreddit, no matter what I put in my flairs filter, it doesn't seem to filter anything out with the flair.

*Are there any related posts in https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues or /r/RESissues?*

this is the most recent one in /r/RESissues -- https://www.reddit.com/r/RESissues/comments/6iwsly/flair_filter_is_not_working/ it's from over a year old.

This is the most recent one for https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues  --
#3716

*What other browser extensions are installed?*
```
 NooBoss : version 0_1_7_0
 Web Store : version 0_2
 Google Drive : version 14_2
 Stylus (beta) : version 1_5_2
 Octo Mate : version 0_16_5
 BlockTube : version 0_2_9
 Pure Dark : version 1_0_0
 Nimbus Screenshot & Screen Video Recorder : version 8_7_7
 Advanced Font Settings : version 0_67
 Plexius : version 1_7_1
 Sonarr Connect : version 1_5_0
 Session Buddy : version 3_6_3
 Sprucemarks : version 2018_2_10_0
 Reddit Enhancement Suite - Beta : version 5_13_5
 EditThisCookie : version 1_5_0
 Homely : version 1_7_1_1
 Nano Adblocker : version 1_0_0_83
 Tampermonkey BETA : version 4_8_5890
 Nano Defender : version 15_0_0_85
 Google Docs Offline : version 1_7
 Lovely forks : version 3_3_0
 Live editor for CSS, Less & Sass - Magic CSS : version 7_0_7
 Reddit Notifier : version 1_0_6
 NooBox Search By Image : version 1_0_2_3
 The Great Suspender : version 7_0_109
 CryptoTokenExtension : version 0_9_73
 Chrono Download Manager : version 0_10_0
 Cloud Print : version 0_1
 Chromium PDF Viewer : version 1
 Radarr and Sonarr for Plex : version 1_0_2
 GitHub Issue Link Status : version 18_6_14_2005
 Bitwarden - Free Password Manager : version 1_37_0
 Chrome Media Router : version 7018_903_0_0
```

**Comment by larsjohnsen** [comment]
I couldn't replicate the issue. Can you post a screenshot of your flair filter configuration and a page listing where you would expect to work?

**Comment by JourneyOver** [comment]
List of my filters for flairs 
https://i.imgur.com/P8IX3qm.png

link for one of the subreddits listing the "expired" flair tag, that should be filtered.
https://www.reddit.com/r/FreeGameFindings/

https://i.imgur.com/PItpDUe.png

**Comment by larsjohnsen** [comment]
I see that you are specifying multiple subreddits with a comma, which is not supported. Try making it into a regex instead.

**Comment by JourneyOver** [comment]
It used to be supported O_o but it's the same thing with https://www.reddit.com/r/Marvel/ and the "Comics" flair as well, and https://www.reddit.com/r/runescape/?count=50&after=t3_afy6bl with the "MTX" flair and the "CH1" flair for clickerheroes as well..

Also I made it into a regex instead with `/(FreeGameFindings|FreeGamesOnSteam)/i` and even attempted to just do `FreeGameFindings` instead and it still doesn't fix the issue, as the flaired posts still end up showing up.

**Comment by JourneyOver** [comment]
Hmm I actually just noticed something, with the version that is on my daily chrome (which is using the Beta version of RES) vs a clean install with only RES (non beta version) when I type in a subreddit it doesn't pop up the whole `type subreddit` name thing, and the subreddits thing also looks completely different between the two variants as well (the subreddits have the white background thing around them in teh clean install of chrome variant)..

Clean Chrome -- 
https://i.imgur.com/Qd8LBAy.png

Daily Chrome --
https://i.imgur.com/fW1YtdD.png

**Comment by larsjohnsen** [comment]
I think I found a bug which is responsible for the flair not being filtered. I'll take a closer look at it. Anyways, thanks for the report.

**Comment by larsjohnsen** [comment]
I realize now that I was wrong about comma-separation not being supported. Sorry about the confusion.

I don't think there's a issue with your Chrome setup, it's just the bug behaving weirdly. I'll release a fix in a few days.

**Comment by JourneyOver** [comment]
It's all good, and I'm guessing the bug is only in the RES Beta version and that's why I wasn't experiencing it in the clean chrome variant with RES non-beta?

**Comment by JourneyOver** [comment]
Yep just a bug with the beta version of RES, I've moved back to the normal releases for now and things are working fine there.

**Comment by larsjohnsen** [comment]
Fixed by 78b27717296a8e5cdcf32f8392be010fb545356c
